packages/api/src/module/write/email-sender/infrastructure/just-log.email-sender.ts

```diff
@@ -0,0 +1,14 @@
+import { Logger } from '@nestjs/common';
+
+import { EmailMessage, EmailSender } from '@/write/email-sender/application/email-sender';
+
+const logger = new Logger('JustLogEmailSender');
+
+export class JustLogEmailSender implements EmailSender {
+  constructor(private readonly next?: EmailSender) {}
+
+  async sendAnEmail(message: EmailMessage): Promise<void> {
+    logger.log('Email message was sent: ', message);
+    this.next?.sendAnEmail(message);
```

Comment:
I think these two lines should be reordered or you can log before `sending` and after `sent`. You can also consider using fake but more realistic [email testing strategy](https://nodemailer.com/smtp/testing/) e.g. for presentation purposes.

Reply:
You're correct - reordered :) 

Regarding better email testing strategy - I'm also into it. Something like that was the part of the task, but we will move it to another in order to do not stop dependent work. 
I'm not sure if Ethernal or MailTrap etc. is good for test on CI, because if there is something bad (outage for example) on the mail server, the tests of our application may break. So I think we can use some email client (like MailHog), which we can manage in Docker to have full controll over that.